hack/manifests/testing/ingress.yaml

```diff
@@ -3,14 +3,23 @@ apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
   name: parodos-ingress
+  annotations:
+    nginx.ingress.kubernetes.io/rewrite-target: /$2
 spec:
   rules:
   - http:
       paths:
       - pathType: Prefix
-        path: /
+        path: /workflow-service(/|$)(.*)
```

Comment:
Shouldn't this change require also an update or a similar configuration on the UI?

Reply:
I do not think that it's the good way, then your swaggers configs are out of control, each service should have his own ingress.